app/src/main/java/com/example/snaplapse/ImageDetailsFragment.kt

```diff
@@ -36,14 +33,31 @@ class ImageDetailsFragment(var item: ItemsViewModel, private val mList: List<Ite
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
+        // Inflate the layout for this fragment
         val view = inflater.inflate(R.layout.fragment_image_details, container, false)
-        var text = view.findViewById<TextView>(R.id.textView)
+        val text = view.findViewById<TextView>(R.id.textView)
         text.text = resources.getString(R.string.image_details).format(item.text)
         val img: ImageView = view.findViewById(R.id.imageView) as ImageView
         img.setImageResource(item.image)
         img.setOnTouchListener(OnSwipeTouchListener(activity, item, mList, view))
 
-        // Inflate the layout for this fragment
+        val likeButton: ImageButton = view.findViewById(R.id.like_button) as ImageButton
+        likeButton.setOnClickListener {
+            Toast.makeText(activity, "Image liked!", Toast.LENGTH_SHORT).show()
```

Comment:
ree

Reply:
These r supposed to be placeholders but I'll add it lol

Reply:
we fix l8r

Reply:
yea idgaf u can just merge